src/app/components/fireside/post/manage/manage.vue

```diff
@@ -71,6 +71,17 @@
 						</template>
 
 						<div class="list-group list-group-dark">
+							<template v-if="shouldShowPin">
```

Comment:
I think we should show the pin controls, only disable it and have a tooltip explaining why you can't pin this specific post.